src/bindings/c/include/openvino/c/ov_compiled_model.h

```diff
@@ -181,5 +182,16 @@ ov_compiled_model_export_model(const ov_compiled_model_t* compiled_model,
  * @ingroup ov_compiled_model_c_api
  * @param compiled_model A pointer to the ov_compiled_model_t to free memory.
  */
-OPENVINO_C_API(void)
-ov_compiled_model_free(ov_compiled_model_t* compiled_model);
+OPENVINO_C_API(void) ov_compiled_model_free(ov_compiled_model_t* compiled_model);
```

Comment:
Does it mean that we don't have automatic `clang-format` check for this file?

Reply:
We need to fix CMake script. Maybe clang format cannot check this style because these headers are not a part of target. So in this case we can call clang format in this manner:
https://github.com/openvinotoolkit/openvino/blob/master/src/inference/CMakeLists.txt#L170

Reply:
It is not CMakeLists.txt issue, there is below line to cover it:
`add_clang_format_target(${TARGET_NAME}_clang FOR_TARGETS ${TARGET_NAME})`

And I also change it to be:
`add_clang_format_target(${TARGET_NAME}_clang FOR_SOURCES ${SOURCES} ${HEADER})`

It is the same behavior.


I think it should come from .clang_format configure in openvino:
`ColumnLimit: 120`

This line's column is 120, so it will not be added new line by clang-format.

Reply:
So, I mean that if we see that some files aren't check by clang-format, so we need to fix CMake script and include these files

Reply:
yes, I see! Thank your helpful reminding!